README.md

```diff
@@ -40,6 +45,17 @@ DeployGate SDK uses `ContentProvider` to initialize itself so you need to remove
 </application>
 ```
 
+And you also need to add `DeployGate.install()` to your implementation.
+For example, add to your custom application class, content provider, or else.
+
+```java
+DeployGate.install(context, /** forceApplyOnReleaseBuild */ false);
+```
+
+```kotlin
+DeployGate.install(app = context, forceApplyOnReleaseBuild = false)
+```
```

Comment:
```suggestion
```

No, we are not providing Kotlin-friendly syntax like named arguments because SDK is built on Java.

Reply:
Sorry, I have misunderstood.
SDK is didn't supported  to call method with named arguments.